v2.0/alter-sequence.md

```diff
@@ -70,8 +70,45 @@ Next, we'll add another record to the table and check that the new record adhere
 +----+----------+--------------------+
 ~~~
 
+### Set the Next Value of a Sequence
+
+In this example, we're going to change the next value of the example sequence (`customer_seq`). Currently, the next value will be `7` (i.e., `5` + `INCREMENT 2`). We will change the next value to `20`.
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT setval('customer_seq', 20, false);
+~~~
+~~~
++--------+
+| setval |
++--------+
+|     20 |
++--------+
+~~~
+
+{{site.data.alerts.callout_info}}The <code>setval('seq_name', value, is_called)</code> function in CockroachDB SQL mimics the <code>setval()</code> function in PostgreSQL, but it does not store the <code>is_called</code> flag. Instead, it sets the value to <code>val - increment</code> for <code>false</code> or <code>val</code> for <code>true</code>. {{site.data.alerts.end}}
```

Comment:
@jseldess, is this note provide enough information about the difference between CRDB SQL and PostgreSQL?

cc @vilterp

Reply:
More info here: https://github.com/cockroachdb/cockroach/issues/21564